src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const handleTabSelected = tabId => {
+    if (activeTabId !== tabId) {
+      onTabSelected(tabId);
+    }
+  };
+
+  const currentTab = tabs.find(tab => tab.id === activeTabId);
+  const activeTab = currentTab ? currentTab.id : tabs[0].id;
```

Comment:
```suggestion
  const activeTabId = currentTab ? currentTab.id : tabs[0].id;
```
The name is a bit confusing